packages/core/src/external/fhir/parse-bundle.ts

```diff
@@ -8,18 +8,16 @@ const metriportPrefixRegex = new RegExp("Metriport/identifiers/Metriport/", "g")
 
 // Moved here from sqs-to-fhir.ts
 
-export function parseRawBundleForFhirServer(
-  payloadRaw: string,
-  patientId: string,
-  log: typeof console.log
-): Bundle {
+export function parseRawBundleForFhirServer(payloadRaw: string, patientId: string): Bundle {
   let payload: any; // eslint-disable-line @typescript-eslint/no-explicit-any
   if (Config.isSandbox()) {
-    const idsReplaced = replaceIds(payloadRaw);
-    log(`IDs replaced, length: ${idsReplaced.length}`);
-    const placeholderUpdated = idsReplaced.replace(placeholderReplaceRegex, patientId);
-    payload = JSON.parse(placeholderUpdated);
-    log(`Payload to FHIR (length ${placeholderUpdated.length}): ${JSON.stringify(payload)}`);
```

Comment:
just making sure that removing the logs was intentional?

Reply:
it was just logging the string length, not very useful